plugins/targetlocker/dblocker/dblocker.go

```diff
@@ -276,14 +273,14 @@ func (d *DBLocker) Lock(jobID types.JobID, targets []*target.Target) error {
 	if len(targets) == 0 {
 		return nil
 	}
-
-	_, err := d.handleLock(int64(jobID), targetIDList(targets), uint(len(targets)), d.lockTimeout, false)
+	lockTimeout := jobTargetManagerAcquireTimeout + config.LockRefreshTimeout
```

Comment:
Aaah you are adding the refresh timeout here. I am fine with this, can you put a note this is added internally into the API doc (in locker.go) so people know

Reply:
Reverted this, it was probably an experiment leftover.